gox/plan.sh

```diff
@@ -0,0 +1,24 @@
+pkg_origin=core
+pkg_name=gox
+pkg_version=master
```

Comment:
Ahhhh!!!! Although I understand why in this case. Thankfully we will at least have the Habitat release coordinate to get a sense of how old it is. I begrudgingly accept!